src/events.js

```diff
@@ -571,10 +571,7 @@ export const addEventCheckInCompleteForm = (isCheckedIn, checkOutFlag) => {
                 hideAnimation();
                 showTimedNotifications({ title: 'Success', body: 'Participant is checked out.' }, 100000, 1500);
                 setTimeout(() => {
-                    const closeButton = document.querySelector('#biospecimenModal .btn[data-dismiss="modal"]');
-                    if (closeButton) {
-                        closeButton.click();
-                    }
+                    dismissBiospecimenModal();
```

Comment:
Why is this line needed? `showTimedNotifications` calls `dismissBiospecimenModal` and runs before this line.